infra/license/license-checker.ts

```diff
@@ -31,6 +31,129 @@ var args = process.argv.slice(2)
 const inputPathOfLicenseList = args[0];
 const verificationTag = args[1];
 
+const VerificationResultType = {
+  pass: 'No license issue found',
+
+  warnConditionalLicense: 'Additional manual verification is needed',
+  warnNeverChecked: 'Never checked license is detected',
+  warnUnknownLicense: 'License is not found',
+
+  failForbiddenLicense: 'Forbidden license is found',
+  failForbiddenPackage: 'Forbidden package is used'
+} as const;
+type VerificationResultTypeKeys = keyof typeof VerificationResultType;
+
+type VerificationResult = {
+  pkgName: string; pkgLicense: string; verificationResult: VerificationResultTypeKeys;
+};
```

Comment:
(optional) The length of `Verification` seems too long. How about use namespace? And, strictly speaking, `VerificationResultType` is not a type but object. So, `Type` suffix is unnecessary.

```typescript
namespace LicenseVerification // or LicenseCheck or something
{

type ResultMessage = {
    [result: string]: {message: string}
}
// Then, you would access the message with `object[result].message`.

const resultMessage: ResultMessage = {
  pass: {message : 'No license issue found'},

  warnConditionalLicense: {message: 'Additional manual verification is needed'},
  warnNeverChecked: {message: 'Never checked license is detected'},
  warnUnknownLicense: {message:'License is not found'},

  failForbiddenLicense: {message:'Forbidden license is found'},
  failForbiddenPackage: {message:'Forbidden package is used'}
} as const;

type Result = keyof typeof resultMessage;

type MetaInfo = { // or type Package
  pkgName: string; pkgLicense: string; verificationResult: Result;
};

} // namespace LicenseCheck
```

Reply:
First of all, thanks you for your detailed suggestions :)
If I am wrong or misunderstood, please let me know!

---

> The length of Verification seems too long. How about use namespace?

I totally agree with this... I tried to reduce the length but couldn't find proper word :(
But with namespace, this can be resolved :)
Thanks, I will apply this :D

> And, strictly speaking, VerificationResultType is not a type but object. So, Type suffix is unnecessary
> you could use index signature for better readability.

Then how about following example?

```typescript
const resultTypeDescription = {
  pass: 'No license issue found',

  warnConditionalLicense: 'Additional manual verification is needed',
  warnNeverChecked: 'Never checked license is detected',
  warnUnknownLicense: 'License is not found',

  failForbiddenLicense: 'Forbidden license is found',
  failForbiddenPackage: 'Forbidden package is used'
} as const;

type ResultType = keyof typeof resultTypeDescription;

type Result = {
  pkgName: string; pkgLicense: string; resultType: ResultType;
};
```

I just wanted to relate ResultType and description, not additional things :)
It was because I wanted to remove duplicated descriptions in both comment and source code...!

Reply:
> Then how about following example?

LGTM:)